dbt_integration.py

```diff
@@ -333,8 +333,9 @@ def safe_parse_project(self, reinit: bool = False) -> None:
         try:
             self.parse_project(init=reinit)
             self.write_manifest_artifact()
-        except Exception:
+        except Exception as e:
```

Comment:
nit: should we prefix "parser error" or something similar to give some hint from which operation the error was triggered?

Reply:
or may be not since the intention of the error is to show it in as a diagnostic message.

Reply:
Indeed it is rather a diagnostic error. I think the other operations are more user actions and less about diagnostics